spec/system/batch_process_spec.rb

```diff
@@ -43,28 +43,6 @@
       expect(page).to have_content("Your records have been retrieved from the MetadataCloud. PTIFF generation, manifest generation and indexing happen in the background.")
       expect(BatchProcess.last.file_name).to eq "short_fixture_ids.csv"
     end
-
-    context "deleting a parent object" do
-      before do
-        page.attach_file("batch_process_file", Rails.root + "spec/fixtures/short_fixture_ids.csv")
-        click_button("Import")
-        po = ParentObject.find(16_854_285)
-        po.delete
-        page.refresh
-      end
-
-      it "can still see the details of the import" do
-        within "td.process_id" do
-          expect(page).to have_link(BatchProcess.last.id.to_s, href: "/batch_processes/#{BatchProcess.last.id}")
-        end
-        within "td.count" do
-          expect(page).to have_content('5')
-        end
-        within "td.view" do
-          expect(page).to have_link('View', href: "/batch_processes/#{BatchProcess.last.id}")
-        end
-      end
-    end
```

Comment:
removed this logic from this file because it too references the classes that were on the `td` fields when we were using a boot strap table. our new data table doesn't have these classes so these fail.

the context still needs to be tested of course, so I'm trying to implement it in "spec/datatables/batch_process_datatable_spec.rb" instead.